package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "name-request",
-  "version": "3.0.13",
+  "version": "3.0.14",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "name-request",
-      "version": "3.0.13",
+      "version": "3.0.14",
```

Comment:
I'm not expecting this change (or the dev/devOptional changes below).

Did you update this manually or did you do `npm i`? What version of node are you using?

Reply:
You should be using node v12 or v14 for now.

Reply:
I was using npm i.

Reply:
I updated the package.json version from 3.0.13 to 3.0.14 and `npm i`.

Reply:
Oh, you're right! Looks like you did the right thing.

I'm still puzzled by the dev -> devOptional changes below. Those should have happened when Omid did his `npm i`.

Well, let's go with your code as-is and see what happens :)